flowcraft/generator/process_details.py

```diff
@@ -71,6 +71,22 @@ def procs_dict_parser(procs_dict):
                     arg_msg = "None"
                 else:
                     arg_msg = ", ".join(dict_proc_info[info])
+            elif isinstance(dict_proc_info[info], dict):
+                # this is used for the "directives", which is a dict
+                if len(dict_proc_info[info]) == 0:
```

Comment:
This can be simplified

```suggestion
                if dict_proc_info[info]:
```